docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -321,7 +321,7 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
 |`table`|The table which contains the key value pairs|Yes||
 |`keyColumn`|The column in `table` which contains the keys|Yes||
 |`valueColumn`|The column in `table` which contains the values|Yes||
-|`filter`|The filter to use when selecting lookups, this is is used to create a where clause on lookup population|No| No Used|
+|`filter`|The filter to use when selecting lookups, this is is used to create a where clause on lookup population|No| Not Used|
 |`tsColumn`| The column in `table` which contains when the key was updated|No|Not used|
```

Comment:
Yea so after looking at the code the docs are wrong, it just pulls the entire table again if any entry has been updated since it was last checked. It does not just pull individual entries and update them, it just pulls the entire lookup table again if any record has been updated since it last generated the cache. Not really ideal but hey.

I could fix the docs in another issue or ideally fix the code to only pull the updated entries from the DB saving some processing time and network? Want me to have a go at patching up the code to match the described behaviour? If so will have a look and create another PR if successful. If not will just update the docs to match the behaviour.